nixos/attestation-server/module.nix

```diff
@@ -45,10 +45,40 @@ in
       type = types.path;
     };
 
+    disableAccountCreation = mkOption {
```

Comment:
Could you separate this into three commits? One for adding disablingAccountCreating, one for adding nginx ACME, and one for setting up email config.